lib/initialize_with_config.rb

```diff
@@ -64,7 +64,7 @@ def call(*)
   end
 
   def default_config
-    {}
+    HashWithIndifferentAccess.new
```

Comment:
What's the difference between the two?

Reply:
HashWithIndifferentAccess doesn't care if the key is a symbol or a string. {} does.